src/Ide/Plugin/Fourmolu.hs

```diff
@@ -47,7 +47,10 @@ provider _lf ideState typ contents fp _ = do
           let p = D.sPgm_F $ D.settings df
           in  if null p then [] else ["-pgmF=" <> p]
         pm = map (("-fplugin=" <>) . moduleNameString) $ D.pluginModNames df
-        ex = map (("-X" <>) . show) $ S.toList $ D.extensionFlags df
```

Comment:
This'll do for now. Really, we shouldn't have to rely on `show` here anyway...

It actually looks like it wouldn't take much of a refactoring to allow passing the `DynFlags` directly. I'll see if I can get this fixed upstream in Ormolu.